src/mapOfPeople.js

```diff
@@ -40,7 +40,15 @@
  * @return {Map}
  */
 function mapOfPeople(people) {
-  // write code here
+  const result = new Map();
+
+  people.forEach((person, index) =>
```

Comment:
Like Misha said - it's not a good idea to use forEach if we are going to create something new and then use it.